layers/core_checks/cc_device_memory.cpp

```diff
@@ -1253,12 +1244,30 @@ bool CoreChecks::PreCallValidateMapMemory2KHR(VkDevice device, const VkMemoryMap
                                      pMemoryMapInfo->offset);
                 }
 
-                if (pMemoryMapInfo->size != VK_WHOLE_SIZE) {
+                if (pMemoryMapInfo->size != VK_WHOLE_SIZE && pMemoryMapInfo->size != mem_info->allocate_info.allocationSize) {
                     skip |= LogError("VUID-VkMemoryMapInfoKHR-flags-09572", pMemoryMapInfo->memory, info_loc.dot(Field::size),
                                      "(0x%" PRIx64
-                                     ") is not VK_WHOLE_SIZE while VK_MEMORY_MAP_PLACED_BIT_EXT is set and "
-                                     "memoryMapRangePlaced is not enabled.",
-                                     pMemoryMapInfo->size);
+                                     ") is not VK_WHOLE_SIZE or the size of memory (%" PRIu64
+                                     ") while VK_MEMORY_MAP_PLACED_BIT_EXT is set and memoryMapRangePlaced is not enabled.",
+                                     pMemoryMapInfo->size, mem_info->allocate_info.allocationSize);
+                }
+            }
+
+            if (pMemoryMapInfo->size == VK_WHOLE_SIZE) {
+                if (mem_info->allocate_info.allocationSize % phys_dev_ext_props.map_memory_placed_props.minPlacedMemoryMapAlignment != 0) {
+                    skip |= LogError("VUID-VkMemoryMapInfoKHR-XXXX", pMemoryMapInfo->memory, info_loc.dot(Field::size),
```

Comment:
```suggestion
                    // Being worked on in https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/6616
                    skip |= LogError("UNASSIGNED-VkMemoryMapInfoKHR", pMemoryMapInfo->memory, info_loc.dot(Field::size),
```

This is how I do it as it helps be track these later